app/assets/stylesheets/themes/scinote.scss

```diff
@@ -62,12 +62,62 @@ table {
 
 #content-wrapper {
   margin-top: 50px;
+  margin-left: 83px;
 
   &.alert-shown {
     margin-top: 102px;
   }
 }
 
+.menu-bar {
```

Comment:
Maybe you can put the css for `menu-bar` into `app/stylesheets/themes/menu_bar.scss` file, so we have it separated.

And another point: I know we said we're not dealing much with responsiveness, but I'd still like the main elements of sciNote (like navigation) to behave semi-responsive. For small heights, the menu bar behaves funky, I'd add the following media query to also handle small heights:
```css
@media(max-height:480px) {
  .menu-bar > ul.nav-bottom {
    position: relative;
    width: auto;
  }
}
```